beluga/include/beluga/sensor/likelihood_field_model.hpp

```diff
@@ -130,20 +130,26 @@ class LikelihoodFieldModel : public Mixin {
    */
   [[nodiscard]] weight_type importance_weight(const state_type& state) const {
     const auto transform = grid_.origin().inverse() * state;
+    const auto x_offset = transform.translation().x();
+    const auto y_offset = transform.translation().y();
+    const auto cos_theta = transform.so2().unit_complex().x();
+    const auto sin_theta = transform.so2().unit_complex().y();
+    const auto unknown_space_occupancy_prob = 1. / params_.max_laser_distance;
     // TODO(glpuga): Investigate why AMCL and QuickMCL both use this formula for the weight.
     // See https://github.com/Ekumen-OS/beluga/issues/153
+    const auto unknown_space_occupancy_likelihood_cubed =
+        unknown_space_occupancy_prob * unknown_space_occupancy_prob * unknown_space_occupancy_prob;
     return std::transform_reduce(
         points_.cbegin(), points_.cend(), 1.0, std::plus{},
```

Comment:
@glpuga I saw this before and I figured I would ask. Why do we add up from 1.0? Shouldn't it be 0.0?

Reply:
I agree that would be the logical thing to do, but [that's not what Nav2 AMCL does](https://github.com/Ekumen-OS/beluga/pull/149#discussion_r1152578137).